SConstruct

```diff
@@ -3398,7 +3398,7 @@ class DXXCommon(LazyObjectConstructor):
 				'variable': EnumVariable,
 				'arguments': (
 					('host_endian', None, 'endianness of host platform', {'allowed_values' : ('little', 'big')}),
-					('host_platform', 'linux' if sys.platform == 'linux2' else sys.platform, 'cross-compile to specified platform', {'allowed_values' : ('win32', 'darwin', 'linux')}),
+					('host_platform', 'linux' if sys.platform == 'linux2' else sys.platform, 'cross-compile to specified platform', {'allowed_values' : ('win32', 'darwin', 'linux', 'openbsd6')}),
```

Comment:
At first look, this change seems useless, since you added no handling of `host_platform` anywhere else, so all later code treats it as unknown.

By chance, all the uses you likely care about test for `!windows` and/or `!darwin`, leaving any unrecognized value as `linux` (which in practice is "Linux and things similar to Linux", hence why it worked for you).

I suspect you only did this because `sys.platform` is `openbsd6` for you, so the default value was not on the `allowed_values` list, causing an SCons error.  Is my interpretation correct?

Reply:
Yes, you are correct. This change got me past the configure error that the platform was not on the list. So far this is what helps get this to build on OpenBSD - sounds like there would be a better way, but I don't understand the configuration well enough for that.

Reply:
For now, all that is missing is a comment explaining it.  I can add that.  My concern is that you are relying on `LinuxPlatformSettings` producing the right results for a non-Linux platform, so without such a comment, or possibly even with it, continued success depends on `LinuxPlatformSettings` not doing anything that's wrong for OpenBSD6.  On the other hand, since those results are right for you, for now, it's minimally disruptive to allow that quirk.